sharepoint/sharepoint-ps/sharepoint-online/Get-SPORestrictedAccessForSitesInsights.md

```diff
@@ -0,0 +1,189 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-sporestrictedaccessforsitesinsights
+applicable: SharePoint Online
+title: Get-SPORestrictedAccessForSitesInsights
+schema: 2.0.0
+author: ishwarit
+ms.author: itambakhe
+ms.reviewer:
+manager:
+---
+
+# Get-SPORestrictedAccessForSitesInsights
+
+## SYNOPSIS
+
+This cmdlet enables the administrator to check status of all active and available reports about insights on sites protected and access denials by restricted access control.
+
+## SYNTAX
+
+```powershell
+Get-SPORestrictedAccessForSitesInsights -RACProtectedSites [-ReportId <Guid>] [-Action <ActionType>] [-InsightsSummary <Boolean>]
+```
+
+```powershell
+Get-SPORestrictedAccessForSitesInsights -ActionsBlockedByPolicy [-ReportId <Guid>] [-Content <ContentType>] [-Action <ActionType>]
+```
+
+## DESCRIPTION
+
+If this cmdlet is executed with `-RACProtectedSites` as parameter, it displays the status of all the active and completed reports.
+
+If this cmdlet is executed with `-RACProtectedSites` `-ReportId` as parameter, top 100 sites with the highest page views that are protected by restricted access control will be displayed.
+
+If this cmdlet is executed with `-RACProtectedSites` `-ReportId` `-InsightsSummary` as parameter, the count of sites protected with restricted access control compared to total number of sites will be displayed.
+
+If this cmdlet is executed with `-ActionsBlockedByPolicy` as parameter, it displays the status of all active and completed reports.
+  
+If this cmdlet is executed with `-ActionsBlockedByPolicy` `-ReportId` `-Content TopSites` as parameter, top 100 sites with the highest access denials by restricted access control will be displayed.
+
+If this cmdlet is executed with `-ActionsBlockedByPolicy` `-ReportId` `-Content TopUsers` as parameter, top 10 users with the highest access denials by restricted access control will be displayed.
+
+If this cmdlet is executed with `-ActionsBlockedByPolicy` `-ReportId` `-Content AllDenials` as parameter, most recent 100 access denials by restricted access control in the last 28 days will be displayed.
+
+If this cmdlet is executed with `-ActionsBlockedByPolicy` `-ReportId` `-Content SiteDistribution` as parameter, distribution of access denials by restricted access control on different site types will be displayed.
```

Comment:
the right place for these is in the parameters section.

this "DESCRIPTION" section is for a summary description of the cmdlet

Reply:
Would like to keep it this way to help in showcasing the parameter combinations.